grpc-proto/proto/schema.proto

```diff
@@ -85,6 +85,11 @@ message CqlKeyspace {
 
 message DescribeKeyspaceQuery {
   string keyspace_name = 1;
+  // The value of `CqlKeyspaceDescribe.hash` the last time the client described that keyspace.
+  // - if it matches the current hash on the server, the server will return an empty response, to
+  //   indicate that the client can continue using the last known value.
+  // - if it doesn't match, or is not provided, the server will return a regular response.
+  google.protobuf.Int32Value hash = 2;
```

Comment:
We could return hash value for otherwise empty case too? But I guess that would prevent returning a pre-constructed empty response so that's perhaps why we don't do that.